spec/models/user_epec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
```

Comment:
Thank you for your dedication and commitment into comprehensively testing the models and their validations. 🚀 

[OPTIONAL] To ensure that all your model validation specs are run successfully.

- Please add the `rspec` gem in the `Gemfile`.
- It would be best to fix the typo `user_epec.rb` to make sure Rspec recognizes the test files.

[REQUIRED] Kindly make sure that all the tests pass to complete the milestone.

![image](https://github.com/NalianAchei2020/rail_blog_app/assets/38088818/379ae410-7a80-499c-baed-ad64c55ac8c5)

> If the tests pass on your end, feel free to reply with a screenshot of the passing tests.

### Summary

🎯 Overall, you have displayed great knowledge in testing different validations.